.github/workflows/test.yml

```diff
@@ -9,8 +9,21 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: jtdor/build-deb-action@v1.0.0
-        with:
-          docker-image: debian:unstable-slim
-          buildpackage-opts: --build=binary --no-sign
+      # FIXME massive hack, just trying to see if this works
+      - name: build and test
+        run: |
+          set -ex
+          git fetch --all
+          git branch pristine-tar origin/pristine-tar
+          git branch upstream/latest origin/upstream/latest
+          CONTAINER_ID="$(docker run --rm --privileged --detach -e DEBIAN_FRONTEND=noninteractive -v "$GITHUB_WORKSPACE:/github/workspace" --workdir=/github/workspace debian:stable-slim tail -f /dev/null)"
```

Comment:
Since the upload is targeting `unstable` (unless you're working on a backport), I think you should use `debian:unstable-slim`, or at least `debian:testing-slim` as the base image.  Otherwise the `apt-get build-dep` step will be pulling build-deps from stable, which won't do you any favors.

Reply:
Yeah great catch. I initially thought I could get away with stable because I was using sbuild + schroots anyway, so I'd end up with an unstable schroot etc. etc.